pkg/txn/clock/hlc.go

```diff
@@ -136,6 +136,12 @@ func (c *HLCClock) Now() (timestamp.Timestamp, timestamp.Timestamp) {
 	return now, timestamp.Timestamp{PhysicalTime: now.PhysicalTime + int64(c.maxOffset)}
 }
 
+func (c *HLCClock) Get() timestamp.Timestamp {
```

Comment:
Is there any particular reason not to use HLC.Now()?

Reply:
The change itself looks fine -- but WHY do you ever need Get()?   How do you plan to use it?   I am not sure what is a valid use case.

Reply:
Wait, I spoke too soon.   I thought this was a small change about Get(), but not sure why it blown out to whole transaction timestamp.    I am not qualified to say the "change itself looks fine".

Reply:
When you want to use the same timestamp with the same value in different places, I think you should just pass that HLC timestamp around. When you want the current timestamp, just use Now(). 

Could you please provide a detailed use case on that Get() method? Thanks.

Reply:
Already removed the `clock.Get() ` method.